plugins/kotlin/uast/uast-kotlin-base/src/org/jetbrains/uast/kotlin/declarations/KotlinUMethodWithFakeLightDelegate.kt

```diff
@@ -107,8 +107,6 @@ internal class KotlinUMethodWithFakeLightDelegateDefaultAccessor(
 ) : KotlinUMethodWithFakeLightDelegateBase<KtProperty>(original, fakePsi, givenParent),
     KotlinUMethodWithFakeLightDelegateAccessorBase {
     override fun computeAnnotations(annotations: SmartSet<UAnnotation>) {
-        // Annotations on property accessor
-        super.computeAnnotations(annotations)
```

Comment:
I updated the comment below to emphasize what went wrong here. Without user-given property accessor, here the `original` is `KtProperty` and we've visited the property's annotation without use-site, resulting in inapplicable annotation being bound to the default property accessor.